Automattic/jetpack
# Connection: Change the error code returned when a remoteRegister XMLRPC call is executed when the blog is already registered to "already_registered"

When a blog is already registered a remoteRegister call fails with "method not registered" because Jetpack does not register the method at all.
In this patch we're registering a substitute method which returns a different error code so that the failure can be distinguished from other cases where remoteRegister is not registered.

#### Changes proposed in this Pull Request:
* Added "already_registered" as a new error code when a remoteRegister call fails due to the blog being already registered.

#### Jetpack product discussion
pbtFPC-22b-p2#comment-5136

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Connect your Jetpack blog to WPCOM using your account.
2. Execute an unauthorized XMLRPC call using the method `jetpack.remoteRegister` - you should get a "method not registered" error.
3. Checkout PR.
4. Repeat step 2 - you should now get an "already_connected" error.
